src/lib/core/api/data-api.ts

```diff
@@ -321,6 +321,44 @@ export const MosaicResponse = type({
   }),
 });
 
+export type TwoByTwoResponse = TypeOf<typeof TwoByTwoResponse>;
```

Comment:
I think you could reuse MosaicResponse here and union in the extra bits needed - especially since both responses start with the `mosaic` key.

Also it looks like conttable/MosaicResponse should include validation for  `sampleSizeTable` ?

Reply:
Thanks Bob! Updated to reuse MosaicResponse using io-ts `intersection` (not `union`, confusingly).

`sampleSizeTable` can be in a generic MosaicResponse, yes.